samples/java-webmvc/pom.xml

```diff
@@ -90,26 +90,6 @@
             <groupId>org.springframework.data</groupId>
             <artifactId>spring-data-commons</artifactId>
         </dependency>
-        <dependency>
-            <groupId>javax.xml.bind</groupId>
```

Comment:
necessary for jdk9 `java.lang.NoClassDefFoundError: javax/xml/bind/UnmarshalException`
keep it here or create java9 profile

Reply:
Dependencies are back and all test are green.